### ISSUE TITLE
unable to fetch pod metrics for pod podname : no metrics known for pod

### ISSUE THREAD

**Issue by munipravy** [prometheus-operator/prometheus-operator#3658]
I am using Prometheus  Operator and Promether server from this repo.

Looing for : Trying to trigger HPA using Prometheus adaptor.

Error : 1 reststorage.go:160] unable to fetch pod metrics for pod podname: no metrics known for pod 

Need suggestion on Prometheus Adaptor for achieving HPA

**Comment by simonpasquier** [comment]
@s-urbaniak @dgrisonnet looks like an issue for prometheus-adapter?

**Comment by dgrisonnet** [comment]
This sounds like the following metrics-server issue: https://github.com/kubernetes-sigs/metrics-server/issues/629

**Comment by munipravy** [comment]
@dgrisonnet I am facing this issue with kubernetes cluster v 1.17, is this issue cluster version specific ?
If it is not cluster specific, can you please let me know how can I achieve HPA with Prometheus Adaptor ?

**Comment by dgrisonnet** [comment]
@munipravy yes after looking into it again, it seems that the issue I linked is specific to 1.19.

As you problem seems to be related to prometheus-adapter, would you mind opening this issue in https://github.com/DirectXMan12/k8s-prometheus-adapter/ and provide your prometheus-adaptor and hpa configurations? I'll help you with the debug from there.

**Comment by munipravy** [comment]
@dgrisonnet created new issue - Hpa is failing with the following specification. #340
Please help me with the same
